lib/data/authentication_service.dart

```diff
@@ -85,4 +86,13 @@ class AuthenticationService {
     return await FirebaseAuth.instance
         .signInWithCredential(githubAuthCredential);
   }
+
+  launchURLApp() async {
+    const url = 'https://app.hackpsu.org/forgot';
+    if (await canLaunch(url)) {
+      await launch(url, forceWebView: true);
```

Comment:
Removing `, forceWebView: true` got it working for me. I know it would be nice if we had it open up in the app so it feels more integrated but for some Android phones I think it'd be safer to have it open up in an actual browser since only that works for me. Would you be ok with this? If you want we could check if it's iOS or Android and have it open up in the app for iOS and in just a browser for Android.